src/components/data/toolbar/TrashMenu.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @author sriram
+ *
+ * A button with menu items for DS trash operations.
+ *
+ */
+
+import React, { useState } from "react";
+import { useTranslation } from "i18n";
+
+import ids from "../ids";
+import styles from "../styles";
+
+import { build } from "@cyverse-de/ui-lib";
+import {
+    Button,
+    makeStyles,
+    Menu,
+    ListItemIcon,
+    ListItemText,
+    MenuItem,
+} from "@material-ui/core";
+
+import {
+    ArrowDropDown,
+    Delete,
+    DeleteForever,
+    RestoreFromTrash,
+    DeleteSweep,
+} from "@material-ui/icons";
+import TrashMenuItems from "./TrashMenuItems";
+
+const useStyles = makeStyles(styles);
+
+export function TrashMenu(props) {
+    const {
+        baseId,
+        selected,
+        handleDelete,
+        handleRestore,
+        handleEmptyTrash,
+    } = props;
+    const { t } = useTranslation("data");
+    const classes = useStyles();
+
+    const [trashAnchor, setTrashAnchor] = useState(null);
+
+    const onTrashClose = () => {
+        setTrashAnchor(null);
+    };
+
+    const onTrashMenuClick = (event) => {
+        setTrashAnchor(event.currentTarget);
+    };
+
+    return (
+        <>
+            <Button
+                id={build(baseId, "trashButton")}
+                variant="outlined"
+                size="small"
+                disableElevation
+                color="primary"
+                className={classes.button}
+                onClick={onTrashMenuClick}
+                aria-haspopup={true}
+                aria-controls={baseId}
+                startIcon={<Delete />}
+                endIcon={<ArrowDropDown />}
+            >
+                {t("trash")}
+            </Button>
+            <Menu
+                id={(baseId, "trashBtnMenu")}
+                anchorEl={trashAnchor}
+                open={Boolean(trashAnchor)}
+                onClose={onTrashClose}
+                getContentAnchorEl={null}
```

Comment:
I don't see this `getContentAnchorEl` prop listed in MUI's `Menu` API.  This might be a typo?

Reply:
Tthat prop comes `Popover` without that we will get the following warning in console.
`Material-UI: You can not change the default `anchorOrigin.vertical` value 
when also providing the `getContentAnchorEl` prop to the popover component.
Only use one of the two props.
Set `getContentAnchorEl` to `null | undefined` or leave `anchorOrigin.vertical` unchanged.`